BP/history/_BP_fix_char_history_issues.info

```diff
@@ -0,0 +1,117 @@
+	sed -i 's/mystic/mystic_1/g' $f
+	sed -i 's/naive_appeaser/education_diplomacy_1/g' $f
+	sed -i 's/nun/devoted/g' $f
+	sed -i 's/proud/arrogant/g' $f
+	sed -i 's/quick/intellect_good_2/g' $f
+	sed -i 's/robust/physique_good_1/g' $f
+	sed -i 's/scholarly_theologian/education_learning_3/g' $f
+	sed -i 's/skilled_tactician/education_martial_3/g' $f
+	sed -i 's/slothful/lazy/g' $f
+	sed -i 's/slow/intellect_bad_1/g' $f
+	sed -i 's/socializer/reveler_2/g' $f
+	sed -i 's/stutter/stuttering/g' $f
+	sed -i 's/thrifty_clerk/education_stewardship_2/g' $f
+	sed -i 's/tough_soldier/education_martial_2/g' $f
+	sed -i 's/ugly/beauty_bad_1/g' $f
+	sed -i 's/underhanded_rogue/education_diplomacy_2/g' $f
+	sed -i '/vaishya/d' $f
+done
+
+#This was applied to regions which made the most sense; some files were reverted afterwards and redone partially by hand
+for f in ./*.txt; do 
+	echo "$f"
+	sed -i 's/trait = digambara_jain/religion = digambara/g' $f
+	sed -i 's/trait = svetambara_jain/religion = svetambara/g' $f
+	sed -i 's/trait = "svetambara_jain"/religion = svetambara/g' $f
+	sed -i 's/trait = "mahayana_buddhist"/religion = mahayana/g' $f
+	sed -i 's/trait = theravada_buddhist/religion = theravada/g' $f
+	sed -i 's/trait = vajrayana_buddhist/religion = vajrayana/g' $f
+	sed -i 's/trait = vaishnavist_hindu/religion = vaishnavism/g' $f
+	sed -i 's/trait = "vaishnavist_hindu"/religion = vaishnavism/g' $f
+	sed -i 's/trait = shaivist_hindu/religion = shaivism/g' $f
+	sed -i 's/trait = "shaivist_hindu"/religion = shaivism/g' $f
+	sed -i 's/trait = smartist_hindu/religion = smartism/g' $f
+	sed -i 's/trait = "smartist_hindu"/religion = smartism/g' $f
+	sed -i 's/trait = zurvanist_trait/religion = zurvanism/g' $f
+	sed -i 's/trait = "zurvanist_trait"/religion = zurvanism/g' $f
+	sed -i 's/trait = siyavashist/religion = siyavashism/g' $f
+	sed -i 's/trait = "siyavashist"/religion = siyavashism/g' $f
+	sed -i 's/trait = mazdeist/religion = mazdayasna/g' $f
+	sed -i 's/trait = "mazdeist"/religion = mazdayasna/g' $f
+	sed -i 's/neoplatonist/platonism/g' $f
+	sed -i 's/religion = bon/religion = old_bon/g' $f
+	sed -i 's/religion = "bon/religion = "old_bon/g' $f
+	sed -i 's/religion=bon/religion = old_bon/g' $f
+	sed -i 's/religion="bon/religion = "old_bon/g' $f
+done
+
+#Suppress sympathy for other religions
+for f in ./*.txt; do 
+	echo "$f"
+	sed -i 's/trait = sympathy/#trait = sympathy/g' $f
+	sed -i 's/trait = "sympathy/#trait = "sympathy/g' $f
+done
+
+#No Equivalents
+# AKA: Nothing in Vanilla CK3 maps well enough
+# agile, cavalry_leader, erudite, harelip
+# immortal, powerful_voice, stressed, 
+# tall (**not** giant), vegetarian, 
+# wroth - split between impatient and wrathful?
+
+#Unclear Mapping
+# AKA: Not sure if anything in Vanilla CK3 maps well enough
+# conscientious, envious, imposing, inspiring_leader, mountain_terrain_leader
+# perceptive, playful, unyielding_leader
+
+#Split Mappings:
+#kinslayer: which type? Dynastic or familial?
```

Comment:
Depends on the specific character and case, can't give a general answer.

Reply:
Right; we can probably move these incomplete traits to a separate issue then.

Reply:
Closing as #51 provides an issue for dealing with these traits.